src/stdlib/crc.rs

```diff
@@ -0,0 +1,587 @@
+use crate::compiler::prelude::*;
+use crate::value;
+use crc::Crc as CrcInstance;
+
+fn crc(value: Value, algorithm: Value) -> Resolved {
```

Comment:
💭 This is one of those (rare) cases where a `#[proc_macro]` can significantly reduce the boilerplate. We could iterate over `valid_algorithms` and keep pushing:
```
 $algo => CrcInstance::<u8>::new(&crc::$algo)
            .checksum(&value)
            .to_string(),
```

But this is just my observation. Feel free to ignore this comment.